### DISCUSSION TITLE
[Cloud Engineer] Remove or refactor workload components with low or no use

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/patterns#35] [Ideas]
https://docs.aws.amazon.com/wellarchitected/latest/sustainability-pillar/remove-or-refactor-workload-components-with-low-or-no-use.html

Intent: Monitor workload activity to identify changes in utilization of individual components over time. Remove components that are unused and no longer required, and refactor components with little utilization, to limit wasted resources.

**Comment by holanita** [comment]
[Issue created](https://github.com/Green-Software-Foundation/green-software-patterns/issues/61)
